.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Does this prevent yarn from creating a `yarn.lock`?

Reply:
Yes, but you want to use the lock file in CI environments to ensure you're installing the same version of dependencies as you've been developing against.